packages/webpack/src/plugin.js

```diff
@@ -55,10 +62,13 @@ class LavaMoatPlugin {
   /**
    * @param {import('./types.js').LavaMoatPluginOptions} [options]
    */
-  constructor(options = { policy: {} }) {
+  constructor(options = {}) {
     if (!options.lockdown) {
       options.lockdown = lockdownDefaults
     }
+    if (!options.policyLocation) {
+      options.policyLocation = path.join('.', 'lavamoat', 'webpack')
```

Comment:
Do we need the default value generation both here and [here](https://github.com/LavaMoat/LavaMoat/pull/959/files#diff-94c0d6243364db60753823cb2b811194dc7fcba7610f63ec9b5b691353457783R38)? Can we settle for just one?

Reply:
we don't. removing one.